cypress-tests/cypress/e2e/PaymentUtils/Trustpay.js

```diff
@@ -54,6 +54,40 @@ export const connectorDetails = {
         },
       },
     }),
+    PaymentIntentWithShippingCost: {
+      Request: {
+        currency: "USD",
+        shipping_cost: 50,
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "requires_payment_method",
+          shipping_cost: 50,
+          amount: 6500,
```

Comment:
Do we need to hard code this?

Or better, create a `const` for `amount` and use it every where. That way, the probability of getting into errors would reduce...

Reply:
Yes we need to hard code this